CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing to The Sailscasts Docs
+
+We appreciate your interest in contributing to this docs! Your contributions are very valuable to us, and we want to ensure that the process is easy and productive for everyone involved. 
+
+Please take a moment to read the guidelines below before contributing to the project.
+
+## Project Description
+
+This project contains the official documentation of all the products built for The Sailscasts Community.
+
+#### These products includes:
```

Comment:
I don't think we should list the projects as the list will get stale pretty quickly and needs update.

Reply:
Alright